code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -73,4 +73,16 @@
 
 /turf/simulated/floor/carpet/burn_tile()
 	burnt = 1
-	update_icon()
\ No newline at end of file
+	update_icon()
+
+
+
+/turf/simulated/floor/fakespace
+	icon = 'icons/turf/space.dmi'
+	icon_state = "0"
+	floor_tile = /obj/item/stack/tile/fakespace
+	broken_states = list("damaged")
+
+/turf/simulated/floor/fakespace/New()
+	..()
+	icon_state = "[rand(0,25)]"
```

Comment:
do ```icon_state = "[((x + y) ^ ~(x * y) + z) % 25]"``` and the fake carpet icons will match up to what a real space turf would look like for that coordinate :P